packages/flutter/lib/src/material/text_field.dart

```diff
@@ -68,8 +68,21 @@ class TextField extends StatefulWidget {
   /// The [maxLines] property can be set to null to remove the restriction on
   /// the number of lines. By default, it is one, meaning this is a single-line
   /// text field. [maxLines] must not be zero. If [maxLines] is not one, then
-  /// [keyboardType] is ignored, and the [TextInputType.multiline] keyboard type
-  /// is used.
+  /// [keyboardType] is ignored, and the [TextInputType.multiline] keyboard
+  /// type is used.
+  ///
+  /// The [maxLength] property is set to null by default, which means the
+  /// number of characters allowed in the text field is not restricted. If
+  /// [maxLength] is set, it will display a character counter below the field,
```

Comment:
nit: "it" is ambiguous so: a character counter will be displayed below the field